pkg/cli/sql_format_table.go

```diff
@@ -275,7 +275,21 @@ func (p *asciiTableReporter) describe(w io.Writer, cols []string) error {
 		p.table = tablewriter.NewWriter(w)
 		p.table.SetAutoFormatHeaders(false)
 		p.table.SetAutoWrapText(false)
-		p.table.SetBorder(false)
+		var outsideBorders, insideLines bool
+		// The following table border modes are taken from psql.
+		// https://www.postgresql.org/docs/12/app-psql.html
+		switch cliCtx.tableBorderMode {
+		case 0:
```

Comment:
i guess if you wanted to be fancy, you could do

```
outsideBorders = (cliCtx.tableBorderMode & 0b01) != 0
insideBorders = (cliCtx.tableBorderMode & 0b10) != 0
```

you can then get rid of the `v < 0 || v > 4` check, if you want to be postgres-esque and allow any value.